Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk
+WORKDIR /
+ADD target/conter-0.0.1-SNAPSHOT.jar conter-0.0.1-SNAPSHOT.jar
```

Comment:
Codacy flags use of the `ADD` command, rather than the `COPY` command, as an error; they cite [this Docker documentation page](https://docs.docker.com/engine/articles/dockerfile_best-practices/#add-or-copy), which says that `ADD` should only be used for extracting the contents of a tarball into the image.